pkg/cmd/config/init/config_init.go

```diff
@@ -36,6 +38,29 @@ type ConfigErr struct {
 	err       error
 }
 
+type p2pStageConfig struct {
+	DefaultEnvs []string `yaml:"default-envs"`
+}
+
+type initConfig struct {
+	Github struct {
+		Organization string `yaml:"organization"`
+	} `yaml:"github"`
+	Repositories struct {
+		Cplatform string `yaml:"cplatform"` // URL, not path
+		Templates string `yaml:"templates"` // URL, not path
+	} `yaml:"repositories"`
+	ConfigDirectory struct {
+		Directory string `yaml:"directory"` // the directory used for the config file
+		File 	  string `yaml:"file"` // the filename of the config file
+	} `yaml:"config-directory"`
```

Comment:
I removed `path` here in favour of `ConfigDirectory`.
Now, I feel like the `File` is irrelevant, the `File` should always be correct.yaml, and only changes to this file are using a different one in a different folder, not renaming it.
But `path` is a poor name and confused the life out of me